plugin/plugin_test.go

```diff
@@ -4,8 +4,100 @@
 
 package plugin
 
-import "testing"
+import (
+	"context"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
 
-func TestValidate(t *testing.T) {
-	t.Skip()
+	"github.com/drone/drone-go/drone"
+	"github.com/drone/drone-go/plugin/validator"
+	"github.com/sirupsen/logrus"
+)
+
+var noContext = context.Background()
+
+const mockToken = "aec773e677ebc489d4e0193c8b971234"
```

Comment:
nit: I do not see a reference to `mockToken`.

Reply:
Good catch, I ended up not using it.